integration_files/SiFive_E21/sim.py

```diff
@@ -512,10 +512,10 @@ def compare_test_run(test, idx, iss, output_dir, report):
                            '{}.{}'.format(test_name, idx))
     instr_gen = os.path.join(output_dir, 'instr_gen',
                            '{}.{}'.format(test_name, idx))
-    #nb_log = os.path.join(rtl_dir, 'trace_core_nb_load.log')
+    nb_log = os.path.join(rtl_dir, 'trace_core_rf_wrenx_00000000.log')
```

Comment:
So right now the script will work fine for single hart single cores because the hart value "0s following the name of trace logs" is hardwired here right now. Need to update this when multi-hart cores comes into our action.